gapipy/resources/tour/_valid_during_range.py

```diff
@@ -0,0 +1,82 @@
+# pylint: disable=no-member
```

Comment:
It looks like most of the other shared supporting "models" are in `gapipy.models`. There doesn't seem to be any hard rules as to what belongs where, but if feels a bit cleaner to me to have `gapipy.models.valid_during_range` instead of a module with a leading underscore in `gapipy.resources`.

Reply:
@rafikdraoui https://github.com/gadventures/gapipy/commit/0246c04ec26f9e83c4cffb6a4251fd3a9c29869f